README.md

```diff
@@ -0,0 +1,24 @@
+<h3 align="center">vacay quiz</h3>
```

Comment:
Suggestion - You can use #, ##, ###, etc for the headings

Take a look in this cheat sheet - https://www.markdownguide.org/cheat-sheet/

Edit - It looks great that you have centered the headings. I would suggest to make the name of the project in h1, so that it gets the most attention

Reply:
h1, h2 are underlined but I didn't want that, therefore I have used h3